src/sql/workbench/contrib/query/browser/actions.ts

```diff
@@ -106,16 +107,31 @@ export class CopyResultAction extends Action {
 	public static COPYWITHHEADERS_ID = 'grid.copyWithHeaders';
 	public static COPYWITHHEADERS_LABEL = localize('copyWithHeaders', "Copy With Headers");
 
+	public static COPYHEADERS_ID = 'grid.copyHeaders';
```

Comment:
I'd prefer to create a new action for it. also, you should add screenshot/gif to demonstrate the changes.